scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -11,8 +11,17 @@ function get_latest_capm3_release() {
 }
 
 # CAPM3 release version which we upgrade from.
-export CAPM3RELEASE="v0.5.0"
-CAPM3_REL_TO_VERSION="$(get_latest_capm3_release)" || true
+export CAPM3RELEASE="v0.5.2"
+# The if below is a workaround until the v0.5.3 or later version is released
+if get_latest_capm3_release | grep -q 'Already'; then
+    export CAPM3_REL_TO_VERSION=${CAPM3RELEASE}
+else
+    # CAPM3 release version which we upgrade to.
+    export CAPM3_REL_TO_VERSION
```

Comment:
This is not correct, if we find new release we have to fetch that release version and set it properly
```suggestion
    # CAPM3 release version which we upgrade to.
    CAPM3_REL_TO_VERSION="$(get_latest_capm3_release)" || true
    export CAPM3_REL_TO_VERSION
```